docs/tempo/website/guides/instrumentation.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Instrumentation Examples
+---
+
+Check out these blog posts for help instrumenting your favorite languages to work with Tempo.
+
+- [Java](https://grafana.com/blog/2021/02/03/auto-instrumenting-a-java-spring-boot-application-for-traces-and-logs-using-opentelemetry-and-grafana-tempo/)
```

Comment:
(nit?) Perhaps we can specify that this tutorial uses Java auto-instrumentation.

Reply:
good call fixed.  going to leave open til tomorrow though b/c i want annanay to see the comment on the images